sensor_msgs/NavSatFix waypoint
int64 mode
# 0 = Append waypoint to list
# 1 = Add waypoint to front of list
# 2 = Clear list and add waypoint
# 3 = Clear list
---
int64 status